lelar66-recommends.md

```diff
@@ -0,0 +1,6 @@
+s is lelar66-recommends.md file
+it has recommendations for what to do in pasadena with a 4-yr old boy
+places to eat and things to do
+## Recommendations for Pasadena,CA
+### Places to Eat
+### Things to Do
```

Comment:
I'd love to see some live music, or go someplace fun to watch the Cubs/Dodgers game. 🎹 ⚾️ Is there anywhere good for those things?